test/validate.test.js

```diff
@@ -80,7 +80,9 @@ describe('Validate', function ValidatesTests() {
           const validateSpy = sinon.spy(function validate() { return isValid; });
           elem.setState({ validate: validateSpy }, function next() {
             assume(validateSpy).is.called(1);
-            check(name, isValid, wasValid);
+            setTimeout(() => { // Needed to allow state to flush
```

Comment:
Since this runs in node, you can use `setImmediate` instead. ([relevant thread](https://stackoverflow.com/questions/24117267/nodejs-settimeoutfn-0-vs-setimmediatefn))

Reply:
That's the function I was trying to remember, thank you! I knew there was a way, but couldn't recall it.